src/Dialogs/Composer/Dialog.vala

```diff
@@ -87,9 +92,11 @@ public class Tuba.Dialogs.Compose : Adw.Window {
 
 	private void update_commit_button () {
 		var allow = false;
-		foreach (var page in t_pages) {
-			allow = allow || (page.can_publish && page.visible);
-			if (allow) break;
+		if (!working) {
+			foreach (var page in t_pages) {
+				allow = allow || (page.can_publish && page.visible);
+				if (allow) break;
+			}
```

Comment:
This should probably be reverted for the reasons mentioned in the previous comment (commit button sensitivity complexity)

If the whole composer is insensitive then the commit button cannot be activated whether it itself is sensitive or not